src/Model/Nodeinfo.php

```diff
@@ -156,12 +156,16 @@ public static function getServices()
 		return $services;
 	}
 
-	public static function getOrganization($config)
+	public static function getOrganization()
```

Comment:
```suggestion
	public static function getOrganization(\Friendica\Core\Config\Capability\IManageConfigValues $config)
```

Reply:
I usually like to have these long class names (with namespace) in top of the file with a proper `use` statement and then have the source code smaller.

Reply:
Of course, but I can't make fancy code suggestions like these. Feel free to add the namespace in a `use` statement.

Reply:
Fixed this + missing doctag and return type.